android/intltestapp/cpp/ReplProxy.cpp

```diff
@@ -0,0 +1,195 @@
+#include "hermes/Platform/Intl/PlatformIntl.h"
+
+#include <fbjni/fbjni.h>
+#include "hermes/VM/Callable.h"
+
+#define C_STRING(x) #x
+
+using namespace ::facebook;
+using namespace ::hermes;
+
+struct ReplProxy : facebook::jni::HybridClass<ReplProxy> {
+
+    static constexpr auto kJavaDescriptor = "Lcom/facebook/hermes/intltestapp/REPLActivity;";
+
+    std::shared_ptr<vm::Runtime> mRuntime;
+
+    static void registerNatives() {
+        javaClassStatic()->registerNatives({
+                                                   makeNativeMethod("initHybrid",
+                                                           ReplProxy::initHybrid),
+                                                   makeNativeMethod("nativeEvalScript",
+                                                           ReplProxy::nativeEvalScript),
+                                                   makeNativeMethod("nativeCollect",
+                                                           ReplProxy::nativeCollect),
+                                                   makeNativeMethod("nativeHeapStats",
+                                                           ReplProxy::nativeHeapStats)
+                                           });
+    }
```

Comment:
This indentation looks really weird. Can we reuse the same formatting setup as Hermes C++ source?

Reply:
When I import this, I'll run it through our linter.

Reply:
It's the default indentation chosen by the default Android Studio formatting .. It is definitely Wierd ! We need to pick a suitable clang formatting rules .. I'll look into it when making the next change ..

Reply:
I'll run this PR through our internal linters to make sure everything is formatted properly before merging it. But I'm not sure how to enforce those rules externally :(